src/error.c

```diff
@@ -334,3 +334,21 @@ void
 raise_invalid_encoding( const char *message ) {
   raise_error( STUMPLESS_INVALID_ENCODING, message, 0, NULL );
 }
+
```

Comment:
move the definition of the function into the private functions section of the file (the bottom portion)